discord/flags.py

```diff
@@ -528,6 +528,7 @@ def default(cls: Type[Intents]) -> Intents:
         self = cls.all()
         self.presences = False
         self.members = False
+        self.messages = False
```

Comment:
```suggestion
```

As noted in our discord:
The message intent is not a normal intent. It shouldn't be excluded.
You receive messages without the intent too, but with skipped fields.
In my old PR I skipped this on purpose.

Read http://dis.gd/mcfaq for more details.